docpages/install/install-conan.md

```diff
@@ -0,0 +1,52 @@
+\page install-conan Installing with Conan 2.0
```

Comment:
doesn't putting the docs in the same pr as the cmakelists introduce a chicken and egg situation?
when we push the cmakelists to master we make it possible to get it into Conan but we also push up docs for how to install it from Conan, when it isn't available in Conan yet.

perhaps the docs should be separated out to their own pr we can merge after Conan is live.

Reply:
I agree. I will remove the md file from this branch. Create a new one and PR for the doc page.